app/multitenant/dynamo_collector.go

```diff
@@ -141,6 +156,25 @@ func NewDynamoDBCollector(
 		}
 	}
 
+	var memcacheClient *memcache.Client
+	if memcachedHost != "" {
+		var memcacheServers memcache.ServerList
+		memcacheClient = memcache.NewFromSelector(&memcacheServers)
+		memcacheClient.Timeout = memcachedTimeout
+		err := updateMemcacheServers(memcachedHost, memcachedService, &memcacheServers)
+		if err != nil {
+			// REVIEWER: Undecided whether this should exit or not. Thoughts?
+			log.Errorf("Could not set memcache servers: %v", err)
```

Comment:
I've documented this. I agree with the desired behaviour, but I think it should wait for a follow-up, as this PR is already big and I'm keen to make incremental improvements.